cfpb/consumerfinance.gov
# Disabling the Helm ES chart tests

By adding a false value to the elastic search value at the top file we can prevent the 3rd party Elasticsearch chart's test from running. The test for the chart doesn't truly test what we want from our Elasticsearch pod and requires 2 pods to be created to successfully run. Disabling it became necessary  for our custom helm tests to run  as the ES test was failing first causing all other tests to not run. This should now allow the command `helm test <cfgov deployment nam>` to run only the tests we make for our deployments.

Note that this value could be overridden at the `/overrides/` directory in the  values file for any environment should we need to bring it back. 


---


## Additions

- test: false

## Changes

- The ES chart tests will no longer run as a default


## How to test this PR

1. Run the helm-install script to make sure your deployment is updated 2. 
run the command `helm test <your deployment name>`
3. There should be only the smoke test pod running 
## Screenshots

<img width="931" alt="Screen Shot 2022-08-23 at 2 39 00 PM" src="https://user-images.githubusercontent.com/3359736/186237928-9f25a530-8f53-41f1-8248-33a46f7f6178.png">

## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
